content/en/docs/ops/diagnostic-tools/controlz/index.md

```diff
@@ -14,23 +14,22 @@ Istio components are built with a flexible introspection framework which makes i
 of a running component. Components open a port which can be used from a web browser to get an interactive view into the state of the
```

Comment:
Is istiod the only component that this applies to now? If so, should we change the title, description, and first paragraph to be more specific?
```
title: Istiod Introspection
description: Describes how to use ControlZ to get insight into a running istiod component.
...
Istiod is build with a flexible introspection framework, called ControlZ, which makes it easy to inspect and manipulate the internal state
of an istiod instance. Istiod opens a port which can be used from a web browser to get an interactive view into its state,
or via REST for access and control from external tools.

When Istiod starts, a message is logged indicating the IP address and port to connect to in order to interact with ControlZ.
```

Reply:
AFAIK it's true. Thanks.